### ISSUE TITLE
Problems with IBQ code

### ISSUE THREAD

**Issue by HalvesChen** [TencentARC/SEED-Voken#48]
Thank you for your excellent work. I wonder what is the role of self.remap in the IBQ class? Is this necessary? In addition, I found that you calculated logits directly to matrix multiplication, does this need to do l2norm in advance?

**Comment by ShiFengyuan1999** [comment]
Hi @HalvesChen, thanks for your interest in our work. We kept the remap-related codes while building our project based on the VQGAN repository, but remap was not actually used in our case. We do not perform l2 norm on visual features and code embeddings before calculating logits.
